content/docs/get-started-with-neon/using-api-keys.md

```diff
@@ -12,8 +12,94 @@ Once authenticated, a user can create and access Projects and [query Project dat
 
 ## Using API keys as a Neon User
 
-API keys allow users to access Neon application programming interface.
+API keys allow users to access the Neon application programming interface.
 
-An API key provides access to any action available to the user. Currently, API keys cannot be scoped to the specific Projects. Neon users can provision multiple API keys. An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked.
+An API key provides access to any action available to the user.An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked. Neon users can generate multiple API keys.
+
+### Issue a New API Key
+
+Here's how to issue a new API key:
+
+1. Start by logging in to [Neon Console](https://console.neon.tech).
+2. Click on your username in the upper right corner, then click the `Settings` button.
+3. Under `Developer Settings` click on `Generate new API Key`.
+4. Choose a unique name that will help you remember what this key is for. For example, if you want a key for your testing environment, name the key `Test Key`.
```

Comment:
I think this is actually a bad example of naming things..
You can never guess that Test Key is for test environment, WFIW it could be something for trying out the API keys.

Reply:
What do you think of "Testing Environment" as the name for this example key?